Stratis.Bitcoin.Features.Wallet/WalletManager.cs

```diff
@@ -650,7 +646,7 @@ public List<UnspentOutputReference> GetSpendableTransactionsInAccount(WalletAcco
                 res = account.GetSpendableTransactions(this.chain.Tip.Height, confirmations);
             }
 
-            this.logger.LogTrace("(-):*.{0}={1}", nameof(res.Count), res.Count);
+            this.logger.LogTrace("(-):*.Count={1}", res.Count());
```

Comment:
`*.Count={1}"` should be `*.Count={0}"`.
Also, this is a tracing log so we're not too worried about performance here and we're taken the general approach to use `nameof` as it leads to fewer mistakes.

Reply:
You cannot do nameof on a function.

Reply:
ah it's a function now, fair enough.